examples/sph/run_dustysettle_tva.py

```
@@ -510,9 +510,9 @@ def compute_L2_error(z, field, z_ref, field_ref):
 
     trap_func = None
     if hasattr(np, "trapezoid"):
-        trap_func = getattr(np, "trapezoid")
+        trap_func = np.trapezoid
     else:
-        trap_func = getattr(np, "trapz")
+        trap_func = np.trapz  # noqa: NPY201
```

Ruff (0.16.0)

[warning] Use ternary operator `trap_func = np.trapezoid if hasattr(np, "trapezoid") else np.trapz` instead of `if`-`else`-block

(SIM108)